x-pack/plugins/security_solution/public/hosts/components/host_risk_information/index.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  useGeneratedHtmlId,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiText,
+  EuiTitle,
+  EuiLink,
+  EuiBasicTable,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyoutFooter,
+  EuiButton,
+  EuiSpacer,
+  EuiBasicTableColumn,
+} from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { useState } from 'react';
+import { HostRiskSeverity } from '../../../../common/search_strategy';
+import { RISKY_HOSTS_DOC_LINK } from '../../../overview/components/overview_risky_host_links/risky_hosts_disabled_module';
+import { HostRiskScore } from '../common/host_risk_score';
+import * as i18n from './translations';
+
+const tableColumns: Array<EuiBasicTableColumn<TableItem>> = [
+  {
+    field: 'classification',
+    name: i18n.INFORMATION_CLASSIFICATION_HEADER,
+    render: (riskScore?: HostRiskSeverity) => {
+      if (riskScore != null) {
+        return <HostRiskScore severity={riskScore} hideBackgroundColor />;
+      }
+    },
+  },
+  {
+    field: 'range',
+    name: i18n.INFORMATION_RISK_HEADER,
+  },
+];
+
+interface TableItem {
+  range?: string;
+  classification: HostRiskSeverity;
+}
+
+const tableItems: TableItem[] = [
+  { classification: HostRiskSeverity.critical, range: '90 and above' },
+  { classification: HostRiskSeverity.high, range: '70 - 90 ' },
+  { classification: HostRiskSeverity.moderate, range: '40 - 70' },
+  { classification: HostRiskSeverity.low, range: '20 - 40' },
+  { classification: HostRiskSeverity.unknown, range: 'Less than 20' },
+];
+
+export const HOST_RISK_INFO_BUTTON_CLASS = 'HostRiskInformation__button';
+
+export const HostRiskInformation = () => {
+  const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
+  const simpleFlyoutTitleId = useGeneratedHtmlId({
+    prefix: 'HostRiskInformation',
+  });
+
+  let flyout;
+
+  if (isFlyoutVisible) {
+    flyout = (
+      <EuiFlyout
+        ownFocus
+        onClose={() => setIsFlyoutVisible(false)}
+        aria-labelledby={simpleFlyoutTitleId}
+        size={450}
+      >
+        <EuiFlyoutHeader hasBorder>
+          <EuiTitle size="m">
+            <h2 id={simpleFlyoutTitleId}>
+              <FormattedMessage
+                id="xpack.securitySolution.hosts.hostRiskInformation.title"
```

Comment:
As we've got an i18n file, what about we put this along with them?

Reply:
Hmm, I need some clarification here. 

It is cleaner to add copy to i18n files, but the i18n guideline says that  `FormattedMessage` is preferred.
<img width="991" alt="Screenshot 2021-12-20 at 16 45 24" src="https://user-images.githubusercontent.com/1490444/146794292-41ebc2d3-034e-4c54-a094-0b0ed52ee743.png">
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md

So, how seriously should I take the guideline?

Reply:
I see, I would say if there's no existing translation file, we could use FormattedMessage. If there's a translation file, make sense to me to keep them in one place.

Reply:
It works for me. I will update the code.